forge-ai/src/main/java/forge/ai/ComputerUtil.java

```diff
@@ -960,6 +962,11 @@ private static Card chooseCardToSacrifice(final CardCollection remaining, final
             }
         }
 
+        if (source instanceof SpellPermanent && (source.getHostCard().hasKeyword(Keyword.EMERGE) || source.getHostCard().hasKeyword(Keyword.OFFERING))) {
```

Comment:
shouldn't the check be source.isEmerge or something because it should only apply to the Emerge Spell?

Reply:
yea somehow I thought they didn't exist